bootstraps/stage2/spack.in.yaml

```diff
@@ -0,0 +1,52 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  # add package specs to the `specs` list
+  specs:
+  - glibc %gcc@=13.2.0 arch=linux-spack-x86_64 ^linux-headers%gcc@12
+  - binutils %gcc@=13.2.0 arch=linux-spack-x86_64
+  - gcc@13.2.0 %gcc@=13.2.0 ~bootstrap ~graphite +binutils arch=linux-spack-x86_64
+  - gmp +stage1 arch=linux-spack-x86_64
+  # we have no C++ right now, disable
+  - berkeley-db ~cxx ~stl
+  - ncurses ~cxx
+  config:
+    install_root:
+      padded_length: 256
+  view: true
+  concretizer:
+    unify: true
+  packages:
+    all:
+      target: [x86_64]
+      variants: ~stage0 +stage1 ~docs ~doc +binutils
+  compilers:
+  - compiler:
+      spec: gcc@=13.2.0
+      paths:
+        cc: /spack/bootstraps/stage1/.spack-env/view/bin/gcc
+        cxx: /spack/bootstraps/stage1/.spack-env/view/bin/g++
+        f77: null
+        fc: null
+      flags: {}
+      operating_system: spack
+      target: x86_64
```

Comment:
And `target: x86_64` is I think not required, but if it is required, then we should rely on `spack compiler find` to fill out the details.